.rubocop.yml

```diff
@@ -42,3 +42,7 @@ Style/RescueModifier:
 
 Metrics/MethodLength:
   Max: 45
+
+Style/MutableConstant:
```

Comment:
I'd be hesitant to disable this. Why is it needed?

Reply:
used the freeze method for the string constants